backend/Watchdog.Core/Watchdog.Core.BLL/Services/ApplicationService.cs

```diff
@@ -138,5 +138,17 @@ public async Task DeleteApplicationAsync(int appId)
             await _context.SaveChangesAsync();
         }
 
+        public async Task<bool> IsProjectNameValid(string projectName, int organizationId)
+        {
+            if (projectName.Length < 3 || projectName.Length > 50)
+            {
+                return false;
+            }
+
+            return !(await _context.Applications
+                .Include(p => p.Platform)
```

Comment:
Why do you need a Platform here?

Reply:
Sorry. Why did I connect the platforms) Fix